packages/eas-cli/src/graphql/queries/BuildQuery.ts

```diff
@@ -0,0 +1,63 @@
+import gql from 'graphql-tag';
+
+import { QueryBuilder, createFragment } from '../queryTools';
+import { Build } from '../types/Build';
+
+type Filters = Partial<Pick<Build, 'platform' | 'status'>> & {
+  order?: number;
+  limit?: number;
+};
+
+type ArtifactFragmentType = Pick<Build, 'artifacts'>;
+
+export const BuildFragments = {
+  Artifacts: createFragment<Build, ArtifactFragmentType>(
+    'ArtifactsFragment',
+    'fragment ArtifactFragment on Build { artifacts { buildUr } }'
+  ),
+};
+
+export class BuildQuery {
+  static byId(buildId: string) {
+    return new QueryBuilder<Build>('Build', {
+      query: ({ fields }) =>
+        gql(`{
+        builds {
+          byId(buildId: "${buildId}") {
+            ${fields}
+          }
+        }
+      }`),
+      resolve: data => data.builds.byId,
+    });
+  }
+
+  static allForApp(appId: string, filters?: Filters) {
```

Comment:
add return type

Reply:
Maybe we can add an eslint rule for that?

Reply:
Can be, I intentionally didn't specify return types in some `QueryBuilder` methods, because they're too complicated and generic.

But I think here they can be safely added and they shouldn't break anything 🤔